docs/user-guide/customization.rst

```diff
@@ -15,11 +15,38 @@ in a platform specific directory, which you can see the path for by running::
   >>> import hermes_core
   >>> hermes_core.print_config()  # doctest: +SKIP
 
+Using your own :file:`configrc` file
+=====================================
+To maintain your own customizations, you must place your customized :file:`configrc` inside the appropriate configuration folder (Which is based off the operating system you are working from). The `AppDirs module <https://github.com/sunpy/sunpy/blob/main/sunpy/extern/appdirs.py>`_  provided by the `sunpy` package is used to figure out where to look for your configuration file. 
 
-To maintain your own customizations place a copy of the default file into the *first* path printed above.
-Do not edit the default file directly as every time you install or update, this file will be overwritten.
+.. warning::
+    Do not just edit the default file directly as every time you install or update, this file will be overwritten.
```

Comment:
```suggestion
    Do not edit the configrc file directly in the Python package as it will get overwritten  every time you re-install or update the package.
```